scripts/plots/nfcens/exevs_nfcens_wave_grid2obs_plots.sh

```diff
@@ -78,13 +82,13 @@ else
   set -x
   echo ' '
   echo '**************************************** '
-  echo '*** ERROR : NO NFCENS *.stat FILE *** '
+  echo '*** FATAL ERROR: NO NFCENS *.stat FILE *** '
   echo "             for ${VDATE} "
   echo '**************************************** '
   echo ' '
   echo "${MODELNAME}_${RUN} $VDATE $vhour : NFCENS *.stat file missing."
   [[ "$LOUD" = YES ]] && set -x
-  ./postmsg "$jlogfile" "FATAL ERROR : NO NFCENS *.stat file for ${VDATE}"
+  ./postmsg "$jlogfile" "FATAL ERROR: NO NFCENS *.stat file for ${VDATE}"
   err_exit "FATAL ERROR: Did not copy the NFCENS *.stat file for ${VDATE}"
   exit
```

Comment:
@SamiraArdani-NOAA I don't think a missing stat file is supposed to cause a FATAL ERROR for plotting?  Maybe this should be just a WARNING.  Plots should be able to run with a few missing stat files.  

Also, I don't think we're supposed to use jlogfiles unless we have compiled code, which we don't have.  

@AliciaBentley-NOAA @ShelleyMelchior-NOAA for awareness.

Reply:
FATAL ERROR is fine here.  I didn't realize that this block of code is for the case when there are zero stat files found.  You can leave this the way it is.

However, we do need you to remove all references to jlogfile.  Just remove the part that says 

`./postmsg "$jlogfile"`

from this line and we'll be OK.  Remove all references to jlogfile in your code.

Reply:
@PerryShafran-NOAA : Ok, I can change the "FATAL" to "WARNING" and remove the jlogfile.

Reply:
@SamiraArdani-NOAA Please change back to FATAL ERROR.  I was wrong to suggest that it should be WARNING.  Clarify for me that this is a case where the code found zero stat files for plotting.  

Also, remove the .postmsg from the above as well, please.

Thanks!